nxdrive/client/remote_client.py

```diff
@@ -868,6 +868,12 @@ def is_filtered(self, path: str, filtered: bool = True) -> bool:
             return self.dao.is_filter(path)
         return False
 
+    def is_proxy(self, ref: str) -> bool:
```

Comment:
There already is that information in the document properties. See how it was done for versions ([NXDRIVE-1639](https://jira.nuxeo.com/browse/NXDRIVE-1639)) and use the key `isProxy` (bool).